tests/Common/SpanChecker.php

```diff
@@ -2,13 +2,122 @@
 
 namespace DDTrace\Tests\Common;
 
+use DDTrace\Contracts\Span;
 use PHPUnit\Framework\TestCase;
 
 /**
  * @see https://phpunit.de/manual/5.7/en/extending-phpunit.html#extending-phpunit.custom-assertions
  */
 final class SpanChecker
 {
+    /**
+     * Asserts a flame graph with parent child relations.
+     *
+     * @param array $traces
+     * @param array $expectedFlameGraph
+     */
+    public function assertFlameGraph(array $traces, array $expectedFlameGraph)
+    {
+        $flattenTraces = $this->flattenTraces($traces);
+        $actualGraph = $this->buildSpansGraph($flattenTraces);
+        foreach ($expectedFlameGraph as $oneTrace) {
+            $this->assertNode($actualGraph, $oneTrace);
+        }
+    }
+
+    /**
+     * @param [] $spansByFingerPrint
```

Comment:
Just a little nitpick :D 

```suggestion
     * @param array $graph
```